# 지원하지 않는 연산자이기 때문에 오류가 발생
print(3 => 4)